src/app/services/baw-api/api-common.helper.ts

```diff
@@ -0,0 +1,148 @@
+import { Id } from "@interfaces/apiInterfaces";
```

Comment:
is this a test-support file?

All of the validate* functions seem to only be usable in tests. If that's the case we should indicated the intended use for this file. I'd move it to `tests/helpers` or something like that.